huaweicloud/services/hss/resource_huaweicloud_hss_quota.go

```diff
@@ -397,7 +397,13 @@ func resourceQuotaDelete(ctx context.Context, d *schema.ResourceData, meta inter
 	}
 
 	if err = common.UnsubscribePrePaidResource(d, cfg, []string{id}); err != nil {
-		return diag.Errorf("error unsubscribe HSS quota (%s): %s", id, err)
+		// When the resource does not exist, the API will return a `400` status code,
+		// and the response body is as follows:
+		// {"error_code": "CBC.30000067",
+		// "error_msg": "Unsubscription not supported. This resource has been deleted or the subscription to this resource has
+		// not been synchronized to CBC..cse://CSBOrderInterfaceAdapterService/rest/csborderadapterservice/v1/open-api
+		// /customers/0970d7b7d400f2470fbec00316a03560/orders/resources/unsubscribe[error_code]:CBC.30000067"}
```

Comment:
```suggestion
		// /customers/{project_id}/orders/resources/unsubscribe[error_code]:CBC.30000067"}
```
do not explose the account information.

Reply:
fixed.